app/models/ontology/sentences.rb

```diff
@@ -155,7 +155,7 @@ def set_theorem_attributes(theorem, hash)
             theorem.state = 'done'
           else
             status_id = ProofStatus::DEFAULT_OPEN_STATUS
-            theorem.state = 'pending'
+            theorem.state = 'not_started_yet'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.